app/views/claimToAdjustPoa/EnterPoAAmountView.scala.html

```diff
@@ -53,6 +53,14 @@
     else controllers.routes.HomeController.show().url
 }
 
+@partiallyOrFullyPaidPoaExists = @{
```

Comment:
I think these view level function is not required as this can be expressed on the level of view model ? + we can cover this with required UT for the view model ?